### ISSUE TITLE
feature requests: run commands in WSL using WSLtty

### ISSUE THREAD

**Issue by doggy8088** [mintty/wsltty#147]
If I want to run command inside WSL, I have to run  the following commands:

```sh
ubuntu1804 run ls -laF ~
```

or

```sh
ubuntu1804 run "ls -laF ~"
```

If I want to run command inside WSLtty, I have to run the following commands:

```sh
%LOCALAPPDATA%\wsltty\bin\mintty.exe --WSL="Ubuntu-18.04" --configdir="%APPDATA%\wsltty" -e sh -c "ls -laF ~"
```

I think it will be better if this can invoke by `Ubuntu~18.04.bat` with parameters.

Here are some use cases:

```sh
ubuntu~18.04 run ls -laF ~
```

or

```sh
ubuntu~18.04 run "ls -laF ~"
```

or

```sh
ubuntu~18.04 run 'ls -laF ~'
```

or

```sh
ubuntu~18.04 run sh -c 'ls -laF ~'
```

**Comment by mintty** [comment]
Simply append the following parameter placeholders to the command line in WSL.bat:
`%1 %2 %3 %4 %5 %6 %7 %8 %9`
I'll make that change for the next release.

**Comment by Biswa96** [comment]
Be careful! `wsl.exe` and `wslhost.exe` options are changing...

**Comment by doggy8088** [comment]
@Biswa96 I have `C:\Windows\System32\bash.exe`, `C:\Windows\System32\wsl.exe`, `%LOCALAPPDATA%\Microsoft\WindowsApps\ubuntu.exe` and `%LOCALAPPDATA%\Microsoft\WindowsApps\ubuntu1804.exe` in my laptop and I have 3 different distro.  As I know, `C:\Windows\System32\bash.exe` is the oldest distro of WSL.  It shouldn't be used anymore.  The `C:\Windows\System32\wsl.exe` looks like the same as `%LOCALAPPDATA%\Microsoft\WindowsApps\ubuntu.exe`.  Do you know why?  I don't know what's the `C:\Windows\System32\wsl.exe` for.

**Comment by Biswa96** [comment]
I told it to @mintty. Read this release note carefully of Build 18305 https://docs.microsoft.com/en-us/windows/wsl/release-notes#build-18305.

**Comment by doggy8088** [comment]
@Biswa96 I finally understand it by reading [this article](https://docs.microsoft.com/en-us/windows/wsl/wsl-config).  Thanks for your hints.

**Comment by CeruleanSky** [comment]
Any particular reason `%1 %2 %3 %4 %5 %6 %7 %8 %9` was chosen over `%*` ?

**Comment by mintty** [comment]
I wasn't aware of the %* option, does it handle more than 9 parameters?

**Comment by CeruleanSky** [comment]
Yes for example:
`a.bat` with `echo %*` inside:

```
C:\temp>a.bat 1 2 3 4 5 6 7 8 9 10 11 12 13 14 etc
1 2 3 4 5 6 7 8 9 10 11 12 13 14 etc

```

**Comment by mintty** [comment]
Released 1.9.7.
